language/move-lang/tests/move_check/locals/use_before_assign_if.exp

```diff
@@ -1,33 +1,27 @@
-error: 
+error[E06002]: use of unassigned variable
+  ┌─ tests/move_check/locals/use_before_assign_if.move:5:17
+  │
+3 │         let x: u64;
+  │             - The variable might not have a value due to this position. The variable must be assigned a value before being used
```

Comment:
The phrasing of ".. might not have a value due to this position" feels a bit awkward/weird. Possible options I see that could read better (feel free to pick something different if you think it will fit better too):
* ".. might not have a value at this position"
* ".. might not have value due to this declaration"
* " .. might not have a value due to this binding"

Reply:
- The first bullet doesn't quite work because it is pointing to were it is used, and this secondary label is why it doesn't have a value
- The other two don't work because the location could have been a `move`. I currently don't distinguish between the reasons why it doesn't have a value

Reply:
Good point on `move`s. Makes sense to keep it this way then (until/unless support for knowing why a variable doesn't have a value is added)